### PR TITLE
Release/v4.94

### PR DESCRIPTION
**Pull request by mperdomo-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "blockchain-wallet-v4",
-  "version": "4.94.18",
```
Comment (Bloodsworth24) [inline package.json:3]: 18

Comment (Bloodsworth24) [inline package.json:3]: Blockchain.com

**Comment by Bloodsworth24** [comment]
> ## Description (optional)
> Add a concise explanation of the changes.
> ## - [ ] - >```
`~~ ~~~~~`
```~
> ## Testing Steps (optional)
> Detail the steps required for the reviewer(s) to verify and test these changes.
> 
> 
**_[]()@mperdomo-bc @jjBlockchain _**

**Comment by Ottos1983** [comment]
> ## Testing Steps (optional)
> Detail the steps required for the reviewer(s) to verify and test these changes.
> 
> 
**_[]()@mperdomo-bc @jjBlockchain _**

**Pull request closed without merging**